diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/.project b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/.project
new file mode 100644
index 0000000..bc486c3
--- /dev/null
+++ b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/.project
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>maplibre_maplibre-react-native</name>
+	<comment>Project maplibre_maplibre-react-native created by Buildship.</comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.jdt.core.javabuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>org.eclipse.buildship.core.gradleprojectbuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.jdt.core.javanature</nature>
+		<nature>org.eclipse.buildship.core.gradleprojectnature</nature>
+	</natures>
+	<filteredResources>
+		<filter>
+			<id>1730963130783</id>
+			<name></name>
+			<type>30</type>
+			<matcher>
+				<id>org.eclipse.core.resources.regexFilterMatcher</id>
+				<arguments>node_modules|\.git|__CREATED_BY_JAVA_LANGUAGE_SERVER__</arguments>
+			</matcher>
+		</filter>
+	</filteredResources>
+</projectDescription>
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/AndroidManifest.xml b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/AndroidManifest.xml
new file mode 100644
index 0000000..cdc05e1
--- /dev/null
+++ b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/AndroidManifest.xml
@@ -0,0 +1,4 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.mapbox.rctmgl">
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
+</manifest>
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/RCTMGLPackage.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/RCTMGLPackage.class
new file mode 100644
index 0000000..ae461a2
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/RCTMGLPackage.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/AbstractEvent.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/AbstractEvent.class
new file mode 100644
index 0000000..893b19d
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/AbstractEvent.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/AbstractEventEmitter.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/AbstractEventEmitter.class
new file mode 100644
index 0000000..a033024
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/AbstractEventEmitter.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/AbstractMapFeature.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/AbstractMapFeature.class
new file mode 100644
index 0000000..25918a4
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/AbstractMapFeature.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/MarkerView.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/MarkerView.class
new file mode 100644
index 0000000..095d5f5
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/MarkerView.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/MarkerViewManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/MarkerViewManager.class
new file mode 100644
index 0000000..a5db8b8
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/MarkerViewManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/RCTMGLCallout.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/RCTMGLCallout.class
new file mode 100644
index 0000000..c0759ca
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/RCTMGLCallout.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/RCTMGLCalloutManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/RCTMGLCalloutManager.class
new file mode 100644
index 0000000..32ee169
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/RCTMGLCalloutManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/RCTMGLMarkerView.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/RCTMGLMarkerView.class
new file mode 100644
index 0000000..710ceda
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/RCTMGLMarkerView.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/RCTMGLMarkerViewManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/RCTMGLMarkerViewManager.class
new file mode 100644
index 0000000..dab4d2b
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/RCTMGLMarkerViewManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/RCTMGLPointAnnotation.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/RCTMGLPointAnnotation.class
new file mode 100644
index 0000000..66c1de2
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/RCTMGLPointAnnotation.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/RCTMGLPointAnnotationManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/RCTMGLPointAnnotationManager.class
new file mode 100644
index 0000000..3ed850c
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/annotation/RCTMGLPointAnnotationManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/CameraStop.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/CameraStop.class
new file mode 100644
index 0000000..21fba25
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/CameraStop.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/CameraUpdateItem.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/CameraUpdateItem.class
new file mode 100644
index 0000000..4113c2b
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/CameraUpdateItem.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/CameraUpdateQueue$OnCompleteAllListener.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/CameraUpdateQueue$OnCompleteAllListener.class
new file mode 100644
index 0000000..8136b97
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/CameraUpdateQueue$OnCompleteAllListener.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/CameraUpdateQueue.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/CameraUpdateQueue.class
new file mode 100644
index 0000000..336a212
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/CameraUpdateQueue.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/RCTMGLCamera.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/RCTMGLCamera.class
new file mode 100644
index 0000000..796c2c7
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/RCTMGLCamera.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/RCTMGLCameraManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/RCTMGLCameraManager.class
new file mode 100644
index 0000000..506605c
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/RCTMGLCameraManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/constants/CameraMode$Mode.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/constants/CameraMode$Mode.class
new file mode 100644
index 0000000..546bae7
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/constants/CameraMode$Mode.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/constants/CameraMode.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/constants/CameraMode.class
new file mode 100644
index 0000000..6ce8f01
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/camera/constants/CameraMode.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/images/RCTMGLImages.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/images/RCTMGLImages.class
new file mode 100644
index 0000000..3f92825
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/images/RCTMGLImages.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/images/RCTMGLImagesManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/images/RCTMGLImagesManager.class
new file mode 100644
index 0000000..478ffa4
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/images/RCTMGLImagesManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/location/LocationComponentManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/location/LocationComponentManager.class
new file mode 100644
index 0000000..943dc65
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/location/LocationComponentManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/location/RCTMGLNativeUserLocation.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/location/RCTMGLNativeUserLocation.class
new file mode 100644
index 0000000..26a3328
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/location/RCTMGLNativeUserLocation.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/location/RCTMGLNativeUserLocationManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/location/RCTMGLNativeUserLocationManager.class
new file mode 100644
index 0000000..19ce70b
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/location/RCTMGLNativeUserLocationManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/LayerSourceInfo.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/LayerSourceInfo.class
new file mode 100644
index 0000000..50b4d2a
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/LayerSourceInfo.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLAndroidTextureMapView.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLAndroidTextureMapView.class
new file mode 100644
index 0000000..c17afcc
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLAndroidTextureMapView.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLAndroidTextureMapViewManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLAndroidTextureMapViewManager.class
new file mode 100644
index 0000000..d15a8b4
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLAndroidTextureMapViewManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLMapView$FoundLayerCallback.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLMapView$FoundLayerCallback.class
new file mode 100644
index 0000000..1899d67
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLMapView$FoundLayerCallback.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLMapView.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLMapView.class
new file mode 100644
index 0000000..455dd1c
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLMapView.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLMapViewManager$MapShadowNode.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLMapViewManager$MapShadowNode.class
new file mode 100644
index 0000000..711908e
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLMapViewManager$MapShadowNode.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLMapViewManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLMapViewManager.class
new file mode 100644
index 0000000..90618b4
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLMapViewManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/helpers/CameraChangeTracker.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/helpers/CameraChangeTracker.class
new file mode 100644
index 0000000..78cda0d
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/mapview/helpers/CameraChangeTracker.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/RCTMGLStyle.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/RCTMGLStyle.class
new file mode 100644
index 0000000..7147f6a
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/RCTMGLStyle.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/RCTMGLStyleFactory.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/RCTMGLStyleFactory.class
new file mode 100644
index 0000000..140b60c
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/RCTMGLStyleFactory.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/RCTMGLStyleFunctionParser$StopConfig.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/RCTMGLStyleFunctionParser$StopConfig.class
new file mode 100644
index 0000000..321f292
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/RCTMGLStyleFunctionParser$StopConfig.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/RCTMGLStyleFunctionParser.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/RCTMGLStyleFunctionParser.class
new file mode 100644
index 0000000..240a581
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/RCTMGLStyleFunctionParser.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/RCTMGLStyleValue.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/RCTMGLStyleValue.class
new file mode 100644
index 0000000..c95e2e6
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/RCTMGLStyleValue.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTLayer.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTLayer.class
new file mode 100644
index 0000000..d2adb84
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTLayer.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLBackgroundLayer.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLBackgroundLayer.class
new file mode 100644
index 0000000..9bec6b1
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLBackgroundLayer.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLBackgroundLayerManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLBackgroundLayerManager.class
new file mode 100644
index 0000000..ce77c6a
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLBackgroundLayerManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLCircleLayer.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLCircleLayer.class
new file mode 100644
index 0000000..97438dc
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLCircleLayer.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLCircleLayerManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLCircleLayerManager.class
new file mode 100644
index 0000000..8fa68af
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLCircleLayerManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLFillExtrusionLayer.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLFillExtrusionLayer.class
new file mode 100644
index 0000000..ccfefc8
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLFillExtrusionLayer.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLFillExtrusionLayerManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLFillExtrusionLayerManager.class
new file mode 100644
index 0000000..1ca854f
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLFillExtrusionLayerManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLFillLayer.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLFillLayer.class
new file mode 100644
index 0000000..34a5d1b
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLFillLayer.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLFillLayerManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLFillLayerManager.class
new file mode 100644
index 0000000..6b133e0
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLFillLayerManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLHeatmapLayer.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLHeatmapLayer.class
new file mode 100644
index 0000000..03b6543
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLHeatmapLayer.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLHeatmapLayerManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLHeatmapLayerManager.class
new file mode 100644
index 0000000..2099f32
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLHeatmapLayerManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLLineLayer.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLLineLayer.class
new file mode 100644
index 0000000..daf2e5e
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLLineLayer.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLLineLayerManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLLineLayerManager.class
new file mode 100644
index 0000000..0270b70
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLLineLayerManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLRasterLayer.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLRasterLayer.class
new file mode 100644
index 0000000..48b4334
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLRasterLayer.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLRasterLayerManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLRasterLayerManager.class
new file mode 100644
index 0000000..771f277
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLRasterLayerManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLSymbolLayer.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLSymbolLayer.class
new file mode 100644
index 0000000..6544aaa
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLSymbolLayer.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLSymbolLayerManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLSymbolLayerManager.class
new file mode 100644
index 0000000..08dd9cf
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/layers/RCTMGLSymbolLayerManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/light/RCTMGLLight.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/light/RCTMGLLight.class
new file mode 100644
index 0000000..807f2b8
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/light/RCTMGLLight.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/light/RCTMGLLightManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/light/RCTMGLLightManager.class
new file mode 100644
index 0000000..0b4cd05
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/light/RCTMGLLightManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLImageSource.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLImageSource.class
new file mode 100644
index 0000000..7ad70ec
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLImageSource.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLImageSourceManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLImageSourceManager.class
new file mode 100644
index 0000000..592ac0c
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLImageSourceManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLRasterSource.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLRasterSource.class
new file mode 100644
index 0000000..44e2418
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLRasterSource.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLRasterSourceManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLRasterSourceManager.class
new file mode 100644
index 0000000..b465658
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLRasterSourceManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLShapeSource.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLShapeSource.class
new file mode 100644
index 0000000..6281a68
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLShapeSource.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLShapeSourceManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLShapeSourceManager.class
new file mode 100644
index 0000000..dba31ec
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLShapeSourceManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLTileSource.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLTileSource.class
new file mode 100644
index 0000000..8a9951c
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLTileSource.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLTileSourceManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLTileSourceManager.class
new file mode 100644
index 0000000..982d8a7
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLTileSourceManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLVectorSource.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLVectorSource.class
new file mode 100644
index 0000000..798941c
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLVectorSource.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLVectorSourceManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLVectorSourceManager.class
new file mode 100644
index 0000000..96a0314
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTMGLVectorSourceManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTSource$OnPressEvent.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTSource$OnPressEvent.class
new file mode 100644
index 0000000..94df3d9
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTSource$OnPressEvent.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTSource.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTSource.class
new file mode 100644
index 0000000..62d5317
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/components/styles/sources/RCTSource.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/AbstractEvent.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/AbstractEvent.class
new file mode 100644
index 0000000..69c25ad
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/AbstractEvent.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/AndroidCallbackEvent.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/AndroidCallbackEvent.class
new file mode 100644
index 0000000..4f66e4a
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/AndroidCallbackEvent.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/EventEmitter.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/EventEmitter.class
new file mode 100644
index 0000000..33bbf88
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/EventEmitter.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/FeatureClickEvent.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/FeatureClickEvent.class
new file mode 100644
index 0000000..51cc6ea
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/FeatureClickEvent.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/IEvent.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/IEvent.class
new file mode 100644
index 0000000..c45e6fa
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/IEvent.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/ImageMissingEvent.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/ImageMissingEvent.class
new file mode 100644
index 0000000..57546a4
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/ImageMissingEvent.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/LocationEvent.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/LocationEvent.class
new file mode 100644
index 0000000..a4a70ce
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/LocationEvent.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/MapChangeEvent.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/MapChangeEvent.class
new file mode 100644
index 0000000..ce2a0c3
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/MapChangeEvent.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/MapClickEvent.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/MapClickEvent.class
new file mode 100644
index 0000000..fc33b83
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/MapClickEvent.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/MapUserTrackingModeEvent.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/MapUserTrackingModeEvent.class
new file mode 100644
index 0000000..0abe3e9
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/MapUserTrackingModeEvent.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/OfflineEvent.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/OfflineEvent.class
new file mode 100644
index 0000000..4e4758e
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/OfflineEvent.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/PointAnnotationClickEvent.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/PointAnnotationClickEvent.class
new file mode 100644
index 0000000..115ca8b
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/PointAnnotationClickEvent.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/PointAnnotationDragEvent.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/PointAnnotationDragEvent.class
new file mode 100644
index 0000000..276a5da
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/PointAnnotationDragEvent.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/constants/EventKeys.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/constants/EventKeys.class
new file mode 100644
index 0000000..3753043
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/constants/EventKeys.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/constants/EventTypes.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/constants/EventTypes.class
new file mode 100644
index 0000000..b5882d4
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/events/constants/EventTypes.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/http/CustomHeadersInterceptor.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/http/CustomHeadersInterceptor.class
new file mode 100644
index 0000000..0e83039
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/http/CustomHeadersInterceptor.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/location/LocationManager$OnUserLocationChange.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/location/LocationManager$OnUserLocationChange.class
new file mode 100644
index 0000000..c7a3ccd
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/location/LocationManager$OnUserLocationChange.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/location/LocationManager.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/location/LocationManager.class
new file mode 100644
index 0000000..2aa8acf
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/location/LocationManager.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/location/UserLocation.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/location/UserLocation.class
new file mode 100644
index 0000000..9f5e2e6
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/location/UserLocation.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/location/UserLocationVerticalAlignment.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/location/UserLocationVerticalAlignment.class
new file mode 100644
index 0000000..626548e
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/location/UserLocationVerticalAlignment.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/location/UserTrackingMode.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/location/UserTrackingMode.class
new file mode 100644
index 0000000..fdd6637
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/location/UserTrackingMode.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/location/UserTrackingState.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/location/UserTrackingState.class
new file mode 100644
index 0000000..26bc20d
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/location/UserTrackingState.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/modules/RCTMGLLocationModule.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/modules/RCTMGLLocationModule.class
new file mode 100644
index 0000000..7ef81bb
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/modules/RCTMGLLocationModule.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/modules/RCTMGLLogging.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/modules/RCTMGLLogging.class
new file mode 100644
index 0000000..bf11d4e
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/modules/RCTMGLLogging.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/modules/RCTMGLModule.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/modules/RCTMGLModule.class
new file mode 100644
index 0000000..15e3151
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/modules/RCTMGLModule.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/modules/RCTMGLOfflineModule.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/modules/RCTMGLOfflineModule.class
new file mode 100644
index 0000000..f5def42
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/modules/RCTMGLOfflineModule.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/modules/RCTMGLSnapshotModule.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/modules/RCTMGLSnapshotModule.class
new file mode 100644
index 0000000..a609156
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/modules/RCTMGLSnapshotModule.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/BitmapUtils.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/BitmapUtils.class
new file mode 100644
index 0000000..60ab898
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/BitmapUtils.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/ClusterPropertyEntry.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/ClusterPropertyEntry.class
new file mode 100644
index 0000000..517b9ab
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/ClusterPropertyEntry.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/ConvertUtils.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/ConvertUtils.class
new file mode 100644
index 0000000..1ad9ba0
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/ConvertUtils.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/DownloadMapImageTask$OnAllImagesLoaded.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/DownloadMapImageTask$OnAllImagesLoaded.class
new file mode 100644
index 0000000..216bce4
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/DownloadMapImageTask$OnAllImagesLoaded.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/DownloadMapImageTask.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/DownloadMapImageTask.class
new file mode 100644
index 0000000..9d5cc49
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/DownloadMapImageTask.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/ExpressionParser.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/ExpressionParser.class
new file mode 100644
index 0000000..325616b
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/ExpressionParser.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/GeoJSONUtils.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/GeoJSONUtils.class
new file mode 100644
index 0000000..1a65353
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/GeoJSONUtils.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/GeoViewport.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/GeoViewport.class
new file mode 100644
index 0000000..14c286c
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/GeoViewport.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/ImageEntry.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/ImageEntry.class
new file mode 100644
index 0000000..a81888f
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/ImageEntry.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/ResourceUtils.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/ResourceUtils.class
new file mode 100644
index 0000000..09a4aea
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/ResourceUtils.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/SimpleEventCallback.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/SimpleEventCallback.class
new file mode 100644
index 0000000..47c6171
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/SimpleEventCallback.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/SphericalMercator.class b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/SphericalMercator.class
new file mode 100644
index 0000000..d70647c
Binary files /dev/null and b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/bin/src/main/java/com/mapbox/rctmgl/utils/SphericalMercator.class differ
diff --git a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/src/main/java/com/mapbox/rctmgl/components/styles/RCTMGLStyleFunctionParser.java b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/src/main/java/com/mapbox/rctmgl/components/styles/RCTMGLStyleFunctionParser.java
index 3aa0522..d66f9ff 100644
--- a/node_modules/@maplibre/maplibre-react-native/android/rctmgl/src/main/java/com/mapbox/rctmgl/components/styles/RCTMGLStyleFunctionParser.java
+++ b/node_modules/@maplibre/maplibre-react-native/android/rctmgl/src/main/java/com/mapbox/rctmgl/components/styles/RCTMGLStyleFunctionParser.java
@@ -3,7 +3,6 @@ package com.mapbox.rctmgl.components.styles;
 import android.util.Log;
 
 import com.facebook.react.bridge.Dynamic;
-import com.facebook.react.bridge.DynamicFromMap;
 import com.facebook.react.bridge.NoSuchKeyException;
 import com.facebook.react.bridge.ReadableArray;
 import com.facebook.react.bridge.ReadableMap;
